torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -28,15 +28,6 @@ std::shared_ptr<SugaredValue> PrintValue::call(
   // print(a, b) is treated as a (a, b) tuple input.
 
   std::vector<Value*> lowered_inputs = toValues(*m.graph(), inputs);
-  if (lowered_inputs.size() == 1 &&
```

Comment:
Ok, I see the move of this code into the lower tuples pass. However, I don't think it does the same thing, because this only lowers the tuple if it is the only argument to prim::Print, while the other one appears to do it universally.